x-pack/test/apm_api_integration/trial/tests/service_maps.ts

```diff
@@ -0,0 +1,262 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { JsonObject } from 'src/plugins/kibana_utils/common';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+
+export default function serviceMapsApiTests({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
```

Comment:
I'll change this to
```suggestion
  const supertestRead = getService('supertestAsApmReadUser');
```

when #70050 is merged
```